packages/postgrest/lib/src/postgrest_builder.dart

```diff
@@ -75,9 +76,7 @@ class PostgrestBuilder<T, S> implements Future<T> {
       isolate: _isolate,
       httpClient: _httpClient,
       options: _options,
-    )
-      .._maybeEmpty = _maybeEmpty
-      .._converter = converter;
+    ).._converter = converter;
```

Comment:
What's the reason behind this removal?

Reply:
Yeah, I just saw a variable being assigned itself here, so I removed it. 

```dart
_maybeEmpty = _maybeEmpty
```

Reply:
It's not assigned to itself. We construct a new `PostgrestBuilder` object here and it doesn't contain the `maybeEmpty` field in the constructor, so we are setting it afterward.

Reply:
You are right. Thanks for this one.